src/modules/addTask.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] You have done well in making the necessary modifications as requested before :+1:. However, please keep in mind that disabling `eslint` in your document should be avoided because `eslint` is a valuable tool that helps to identify errors and enhance the overall quality of your code.